tasks/main.yml

```
@@ -1,17 +1,12 @@
 ---
-# tasks file for openvpn
-
 - name: Load var file based on the OS type
   include_vars: "{{ lookup('first_found', params) }}"
   vars:
     params:
       files:
-        # Add regex_replace() filters to remove spaces and
-        # slashes. This allows the OS family/distribution for Kali,
-        # "Kali GNU/Linux", to be easily mapped to a vars file.
-        - "{{ ansible_distribution | regex_replace('[ /]', '_') }}_{{ ansible_distribution_version }}.yml"
-        - "{{ ansible_distribution | regex_replace('[ /]', '_') }}.yml"
-        - "{{ ansible_os_family | regex_replace('[ /]', '_') }}.yml"
+        - "{{ ansible_distribution }}_{{ ansible_distribution_version }}.yml"
```

Should this also switch? It would require changing `Ubuntu_16.04.yml` as well.
```suggestion
        - "{{ ansible_distribution }}_{{ ansible_distribution_release }}.yml"
```

---

Thanks!  I noticed that this morning too.